packages/html-reporter/src/reportView.css

```diff
@@ -33,8 +33,57 @@ body {
   border-top: 1px solid var(--color-border-default);
 }
 
+.htmlreport {
+  gap: 24px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.htmlreport header {
+  width: 300px;
+  order: 2;
+}
+
+.htmlreport main {
+  width: 100%;
+  order: 1;
+}
+
+.metadata-view a {
+  color: var(--color-accent-fg);
+  text-decoration: none;
+}
+
+.metadata-view a:hover {
+  text-decoration: underline;
+}
+
+.metadata-view h1 {
+  font-size: 16px;
+  font-weight: 600;
+  margin-bottom: 16px;
+  margin-top: 0;
+  line-height: 1.5;
+}
+
 @media only screen and (max-width: 600px) {
   .report {
     padding: 0 !important;
   }
 }
+
+@media only screen and (max-width: 900px) {
```

Comment:
@pavelfeldman To not have anything above search box, should we just `display: none` the header at this point? If you look at the Break Point in the description, the header sticks around for a while so I'd totally be OK with it just being gone when we no longer have room for sidebar.